themes/base.theme.bash

```diff
@@ -98,6 +98,44 @@ function scm_prompt_info {
   [[ $SCM == $SCM_SVN ]] && svn_prompt_info && return
 }
 
+function scm_prompt_status {
```

Comment:
Initially, I had thought about going with the environment variable idea, but I opted for this approach for a couple reasons:

1. While the `git` SCM tool is the only one with a short status at the moment, it could be extended for `hg` and `svn` as well. I imagine their respective functions provide similar details as the `git_prompt_vars` does and would benefit from a faster, shorter formed prompt. But I don't have any of those types of repositories to test with.
2. Adding extra conditional checks will only continually slow down the prompt execution speed.

I think the only thing you gain from using a variable like `SCM_GIT_SHORT_STATUS` is code duplication, often times a trade-off with speed improvements. :) That's because this approach still remains backwards compatible and you won't have to rewrite themes any more than changing the function call you're using. For backwards compatibility, do you prefer to require a change in variables instead of changing a function call? I understand if that's the case.

In reality, some (most?) people probably don't really care for anything but `git` status. At least that's the case for me. :) So ideally, we should just be able to call a `git` specific function for either a detailed or short prompt without requiring extra conditionals to figure out what SCM tool we're working in; albeit, the conditionals for `git` do come first. I'll make sure to change this new git short status form to enable the capability to just be called directly.

Since your primary goal is for maintainability, I'll make the changes you suggest.